tests/integ/test_workflow.py

```diff
@@ -869,3 +871,78 @@ def test_training_job_with_debugger_and_profiler(
             pipeline.delete()
         except Exception:
             pass
+
+
+def test_one_step_data_wrangler_processing_pipeline(
+    sagemaker_session, role, cpu_instance_type, pipeline_name, region_name
+):
+    instance_count = ParameterInteger(name="InstanceCount", default_value=1)
+
+    recipe_file_path = os.path.join(DATA_DIR, "workflow", "dummy_recipe.flow")
+    input_file_path = os.path.join(DATA_DIR, "workflow", "dummy_data.csv")
+
+    output_name = "1bd0aaad-9c93-41b2-8d42-58e214f0843f.default"
+    output_content_type = "CSV"
+    output_config = {output_name: {"content_type": output_content_type}}
+    job_argument = [f"--output-config '{json.dumps(output_config)}'"]
+
+    inputs = [ProcessingInput(input_name="job_data", source=input_file_path, destination="/opt/ml/processing")]
+
+    output_s3_uri = f"s3://{sagemaker_session.default_bucket()}/output"
+    outputs = [
+        ProcessingOutput(
+            output_name=output_name,
+            source="/opt/ml/processing/output",
+            destination=output_s3_uri,
+            s3_upload_mode="EndOfJob",
+        )
+    ]
+
+    data_wrangler_processor = DataWranglerProcessor(
+        role=role,
+        data_wrangler_flow_source=recipe_file_path,
+        instance_count=instance_count,
+        instance_type=cpu_instance_type,
+        max_runtime_in_seconds=86400,
+    )
+
+    data_wrangler_step = ProcessingStep(
+        name="data-wrangler-step",
+        processor=data_wrangler_processor,
+        inputs=inputs,
+        outputs=outputs,
+        job_arguments=job_argument,
+    )
+
+    pipeline = Pipeline(
+        name=pipeline_name,
+        parameters=[instance_count],
+        steps=[data_wrangler_step],
+        sagemaker_session=sagemaker_session,
+    )
+
+    definition = json.loads(pipeline.definition())
+    expected_image_uri = image_uris.retrieve("data-wrangler", region=sagemaker_session.boto_region_name)
```

Comment:
Should we also start a pipeline execution and verify if the execution succeeds so that we have complete coverage in this test?

Reply:
I think this is fine...The pipeline definition is still processing job and nothing else changed. If we do actual execution, it is really just testing dw container.